packages/bicep-deploy-common/src/azure.ts

```diff
@@ -24,7 +24,8 @@ const endpoints = {
 };
 
 export function createDeploymentClient(
```

Comment:
Note a huge deal, but it might be worth putting some thought into what is exposed at the package level, and creating an `index.ts` file.

Otherwise, your imports are going to look like:
```ts
import { DeployConfig } from "@azure/bicep-deploy-common/src/config"
```
Instead of:
```ts
import { DeployConfig } from "@azure/bicep-deploy-common"
```

If you take the suggestion to add the NPM package as a local dependency, this should help with figuring out what this experience should look like.